dev/sparktestsupport/modules.py

```diff
@@ -469,7 +469,7 @@ def __hash__(self):
     name="yarn",
     dependencies=[],
     source_file_regexes=[
-        "yarn/",
+        "resource-managers/yarn/",
```

Comment:
You also need to update L477 (and probably a similar change in the mesos entry).